numpy/core/tests/test_multiarray.py

```diff
@@ -8007,6 +8007,35 @@ def test_ctypes_data_as_holds_reference(self, arr):
         break_cycles()
         assert_(arr_ref() is None, "unknowable whether ctypes pointer holds a reference")
 
+    def test_ctypes_as_parameter_holds_reference(self):
+        arr = np.array([None]).copy()
+
+        arr_ref = weakref.ref(arr)
+
+        ctypes_ptr = arr.ctypes._as_parameter_
+
+        # `ctypes_ptr` should hold onto `arr`
+        del arr
+        break_cycles()
+        assert_(arr_ref() is not None, "ctypes pointer did not hold onto a reference")
+
+        # but when the `ctypes_ptr` object dies, so should `arr`
+        del ctypes_ptr
+        break_cycles()
+        assert_(arr_ref() is None, "unknowable whether ctypes pointer holds a reference")
+
+    def test_ctypes_data_as_no_circular_reference(self):
```

Comment:
Instead of adding a new test, it sounds like you can just remove the second `break_cycles()` call in the `test_ctypes_data_as_holds_reference` test

Reply:
That seems right.